alerts/charts/templates/certmanager.yaml

```
@@ -5,7 +5,7 @@
 apiVersion: cert-manager.io/v1
 kind: Issuer
 metadata:
-  name: {{ .Release.Namespace }}-prometheus-issuer
+  name: '{{ .Release.Namespace }}-prometheus-issuer'
```

Is this naming better? 
```suggestion
  name: '{{ .Release.Namespace }}-alerts-issuer'
```

---

https://github.com/cloudoperators/greenhouse-extensions/pull/658/commits/095ed6550303c14d7554c9b01767db4525537f14